install_config/web_console_customization.adoc

```diff
@@ -1014,3 +1014,38 @@ xref:../install_config/install/advanced_install.adoc#advanced-install-configurin
 # See: https://docs.openshift.com/enterprise/latest/install_config/aggregate_logging.html
 #openshift_master_logging_public_url=https://kibana.example.com
 ----
+
+[[configuring-quota-notification-messages]]
+== Configuring Quota Notification Messages
```

Comment:
This section should probably be higher up in the doc, maybe under "Configuring Catalog Categories"?